lesson04/assignment/css/style.scss

```diff
@@ -0,0 +1,52 @@
+
+// insperation source code https://thoughtbot.com/blog/transitions-and-transforms
+
+@mixin flexVar($flexType: center) {
+  display: flex;
+  justify-content: $flexType;
+}
+
+h1 {
+  margin-top: 10%;
+  text-align: center;
+  font-size: 800%;
+}
+
+// Nesting
+.image-box {
+  @include flexVar;
+  align-items: center;
+  flex-direction: column;
+  width: 15%;
+  margin: 1%;
+
+  img {
+    width: 100%;
+    border-radius: 50%;
+  }
+
+  h2 {
+    margin-top: 10%;
+    font-size: 400%;
+    line-height: 0;
+  }
+
+  p {
+    line-height: 0;
+    font-size: 250%;
+  }
+}
+
+.box {
+  @include flexVar;
+  margin: 5%;
+  // Hover off transition
+  transition: all 3s;
+}
+
+// Hover transition/transform scale
+.box:hover {
+  transform: scale(1.5);
+  transition: all 3s;
```

Comment:
Since the transition is applied to the .box class, you shouldn't need to include it on the :hover state.